ui/src/layout/Layout.tsx

```diff
@@ -112,9 +98,7 @@ class Layout extends React.Component<
             <MuiThemeProvider theme={theme}>
                 <HashRouter>
                     <div>
-                        {hasNetwork ? null : (
-                            <NetworkLostBanner height={64} retry={this.doReconnect} />
-                        )}
+                        {hasNetwork ? null : <NetworkLostBanner height={64} retry={tryReconnect} />}
```

Comment:
No notification will be displayed when the retry button on the NetworkLostBanner is clicked. It seems like the default value for quiet doesn't work (I dunno why). I got it working by explicitly passing `false` to the tryAgain method.
```suggestion
                        {hasNetwork ? null : <NetworkLostBanner height={64} retry={() => tryReconnect(false)} />}
```

Reply:
Not really necessary to explicit the argument on `tryReconnect` function. I've added argument type and called the function without arguments. Now it seems to work.

Reply:
And I fixed the formatting with `yarn format` :)